awx/main/migrations/0081_v360_notify_on_start.py

```diff
@@ -22,7 +37,8 @@ class Migration(migrations.Migration):
             name='notification_templates_started',
             field=models.ManyToManyField(blank=True, related_name='unifiedjobtemplate_notification_templates_for_started', to='main.NotificationTemplate'),
         ),
-#        migrations.RunPython() stuff goes in here
+        # Separate out "any" notifications into "success" and "error" before the "any" state gets deleted.
+        migrations.RunPython(forwards_split_unified_job_template_any, forwards_split_organization_any),
```

Comment:
This is slightly wrong:

https://docs.djangoproject.com/en/2.2/ref/migration-operations/#runpython

You want:

```python
migrations.RunPython(forwards_split_unified_job_template_any, None),
migrations.RunPython(forwards_split_organization_any, None),
```

Reply:
This is getting fixed here: https://github.com/ansible/awx/pull/4098